contracts/AcceleratingDistributor.sol

```diff
@@ -97,9 +97,14 @@ contract AcceleratingDistributor is Testable, ReentrancyGuard, Pausable, Ownable
         uint256 maxMultiplier,
         uint256 secondsToMaxMultiplier
     ) public onlyOwner {
+        // Validate input to ensure system stability and avoid unexpected behavior.
         // Because of the way balances are managed, the staked token cannot be the reward token. Otherwise, reward
         // payouts could eat into user balances.
         require(stakedToken != address(rewardToken), "Staked token is reward token");
+        require(maxMultiplier > 1e18, "maxMultiplier be larger than 1");
+        require(maxMultiplier < 1000000000e18, "maxMultiplier can not be set too large");
```

Comment:
Just based on back-of-the-envelope math, a reasonable max limit to this IMO would be `1e36`. This would allow it to safely be multiplied by all numbers 1e18 and below (as was mentioned in the audit report). It would also allow it to safely be multiplied by an emission as large as 1e18 on an 18 decimal coin.